### PR TITLE
Clean up embedding code

### PR DESCRIPTION
**Pull request by WonyoungChoi**
### C#
- Change private properties to private fields.
- Fix class layout orders. [SA1201](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1201.md) , [SA1202](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1202.md)

- Remove `DartEntrypoint` property of `FlutterApplication` and `FlutterServiceApplication`.
- Simplify `NUIFlutterView.RunEngine()` .

### C++
- Remove unused private member `dart_entrypoint_args_` from `FlutterApp` and `FlutterServiceApp`.

### CODE REVIEW

File: embedding/csharp/Tizen.Flutter.Embedding/ElmFlutterView.cs [hunk 1]
```diff
@@ -30,40 +30,37 @@ protected override IntPtr CreateHandle(EvasObject parent)
     /// </summary>
     public class ElmFlutterView
     {
+        private int _initialWidth;
+        private int _initialHeight;
+        private EvasObject _parent;
+        private FlutterDesktopView _desktopView;
```
[the remark this answers is not in the document]
Comment (swift-kim) [reply embedding/csharp/Tizen.Flutter.Embedding/ElmFlutterView.cs:36]: Please also notice that `FlutterApplication` and `FlutterServiceApplication` (and their C++ counterparts) also have `View` or `view_` as a member.

File: embedding/csharp/Tizen.Flutter.Embedding/FlutterApplication.cs [hunk 2]
```diff
@@ -119,11 +114,20 @@ public override void Run(string[] args)
             base.Run(args);
         }
 
+        public FlutterDesktopPluginRegistrar GetRegistrarForPlugin(string pluginName)
```
[the remark this answers is not in the document]
Comment (WonyoungChoi) [reply embedding/csharp/Tizen.Flutter.Embedding/FlutterApplication.cs:116]: It's recommended. https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1202.md

**Comment by WonyoungChoi** [comment]
> I mean, if you're to delete any comments, they should be removed from the C++ embedding as well. If you don't want to do that, the comments should be kept in the C# code. Note that some comments may contain information that is important to users.

I just remove the documentation comments of the private members in C#. Keeping the comments is also possible. How about you?

**Comment by swift-kim** [comment]
If you're not sure, you should keep them as they are.

File: embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs [hunk 3]
```diff
@@ -58,29 +56,21 @@ public bool RunEngine()
                 return false;
             }
 
-            if (Engine == null)
-            {
-                Engine = new FlutterEngine();
-            }
-
+            Engine = Engine ?? new FlutterEngine();
```
Comment (swift-kim) [inline embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs:59]: nit: There's another occurrence of this in `ElmFlutterView.cs`.

**Review by swift-kim — COMMENTED** [review]
Thank you!

File: embedding/csharp/Tizen.Flutter.Embedding/ElmFlutterView.cs [hunk 4]
```diff
@@ -31,39 +31,51 @@ protected override IntPtr CreateHandle(EvasObject parent)
...
         /// </summary>
-        private FlutterDesktopView View { get; set; } = new FlutterDesktopView();
+        private int _initialHeight;
 
         /// <summary>
-        /// The backing Evas object for this view.
-        /// </summary>
-        public EvasObject EvasObject { get; private set; } = null;
+        /// The parent of <see cref="EvasObject"/>.
+        /// </summary>
```
[the remark this answers is not in the document]
Comment (WonyoungChoi) [reply embedding/csharp/Tizen.Flutter.Embedding/ElmFlutterView.cs]: Done. thank you.

File: embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs (hunk 3 above)
Comment (WonyoungChoi) [reply embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs:59]: Oh I missed. fixed it.

File: embedding/csharp/Tizen.Flutter.Embedding/FlutterServiceApplication.cs [hunk 5]
```diff
@@ -20,20 +19,15 @@ public class FlutterServiceApplication : ServiceApplication, IPluginRegistry
         public string DartEntrypoint { get; set; } = string.Empty;
 
         /// <summary>
-        /// The list of Dart entrypoint arguments.
+        /// Whether the app has started.
         /// </summary>
-        private List<string> DartEntrypointArgs { get; } = new List<string>();
```
Comment (JSUYA) [inline embedding/csharp/Tizen.Flutter.Embedding/FlutterServiceApplication.cs:25]: I have a question. @swift-kim 
Application doesn't originally have a scenario to set Dart engtrypoint argument?
Couldn't it be opened to change the private property to public and set it the same as DartEntryPoint?

File: embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs [hunk 6]
```diff
@@ -58,29 +56,21 @@ public bool RunEngine()
...
 
             Size2D size = GetDefaultSize();
-            Type baseType = typeof(NativeImageQueue).BaseType.BaseType.BaseType;
-            FieldInfo field = baseType.GetField("swigCPtr", BindingFlags.NonPublic | BindingFlags.Instance);
-            NativeImageQueue nativeImageQueue =
+
+            var nativeImageQueue =
                 new NativeImageQueue((uint)size.Width, (uint)size.Height, NativeImageQueue.ColorFormat.RGBA8888);
-            HandleRef nativeImageQueueRef = (HandleRef)field.GetValue(nativeImageQueue);
+            var nativeImageQueueRef = GetFieldValue<HandleRef>(nativeImageQueue, typeof(Tizen.NUI.Disposable), "swigCPtr");
```
Comment (JSUYA) [inline embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs:70]: Wow Thanks 👍

File: embedding/csharp/Tizen.Flutter.Embedding/FlutterServiceApplication.cs (hunk 5 above)
Comment (swift-kim) [reply embedding/csharp/Tizen.Flutter.Embedding/FlutterServiceApplication.cs:25]: As we decided not to implement the entrypoint arguments in https://github.com/flutter-tizen/flutter-tizen/pull/387, the property seems no longer necessary. If you want to allow manually passing the arguments from the runner code (I don't think there will be such use case), we can make the property public.

**Review by swift-kim — APPROVED** [review]
If you'd like to publish this change, please increase the nuget package version as well.

**Comment by swift-kim** [comment]
@WonyoungChoi Did you read my comment? Are we not going to publish this kind of changes?

**Pull request merged**